docs/navigation.rb

```diff
@@ -36,6 +36,14 @@ def without_prefix(prefix)
         path: 'Metasploit-Guide-Post-Gather-Modules.md',
         title: without_prefix('Metasploit Guide ')
       },
+      {
+        path: 'Attacking-ADCS-ESC-Vulnerabilities.md',
+        title: 'Attacking AD CS ESC Vulnerabilities Using Metasploit'
+      },
+      {
+        path: 'Setting-Up-A-AD-CS-Target.md',
+        title: 'Setting up a Vulnerable AD CS Target'
+      },
```

Comment:
After a rebase can we slot this content into the AD CS folder that was added 👀 

Setting up a lab environment probably belongs in the Overview page

Reply:
This should be fixed now.